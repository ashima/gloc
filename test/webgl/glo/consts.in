const float g = 0.2;

vec2 myfun(vec2 m) {
     const vec2 f = vec2(g);
     return f+m;
}
