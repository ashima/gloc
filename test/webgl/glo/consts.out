{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "author": {},
    "license": [ "Copyright (C) 2011 . All rights reserved.", "" ],
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [ "g", "f" ],
      "outsym": [ "myfun" ],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source":
        "#line 1 GLOC_0\nconst float g = 0.2;\n\nvec2 myfun(vec2 m) {\n     const vec2 f = vec2(g);\n     return f+m;\n}"
    }
  ],
  "linkmap": {}
}
